src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1044,10 +1049,9 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
         List<String> possibleReceiversWithoutResponsesForGiver = new ArrayList<>();
 
         String prevGiver = "";
-        int responseRecipientIndex = 0;
-        int responseGiverIndex = 0;
-        int userIndex = 0;
-        Map<String, Integer> userIndexesForComments = new HashMap<String, Integer>();
+        List<String> responseGiverRecipientList = new ArrayList<>();
+        // TODO: Revamp implementation of comment indexes for Question View
+        responseGiverRecipientList.add("Dummy Value");
```

Comment:
@xpdavid  I used you second suggestion. In previous implementation, indexes start from `1`, so I had to put this dummy value to make sure their are no other html changes.

Reply:
Also, for the performance consideration, I think we should use Map instead. The `contains` method run in O(n). Also, in doing this, we don't need the magic dummy value?